.github/workflows/ci.yaml

```diff
@@ -378,8 +378,62 @@ jobs:
         with:
           api-key: ${{ secrets.DATADOG_API_KEY }}
 
+  # We don't run the full test-suite for Windows & MacOS, so we just run the CLI tests on every PR.
+  # We run the test suite in test-go-pg, including CLI.
```

Comment:
suggestion: add some more context on why we're doing this

Reply:
I've added this to the description of the PR:

> We're still investigating why these OSs are a lot less reliable. It's likely that the VMs on which the builds are run have different characteristics from our Ubuntu runners such as disk I/O, network latency, or something else. In the meantime, we need to start trusting CI failures in `main` again, as the current failures are too noisy / vague for us to correct.

> We've also considered hosting our own runners where possible so we can get OS-level observability to rule out some possibilities.

Is that enough for you, or do you specifically want it in the comment?

Reply:
That's fine 👍